docs/ref/django-admin.txt

```diff
@@ -757,6 +757,9 @@ generated migration files to ``stdout``.
 Merges model changes into the latest migration and optimize the resulting
 operations.
 
+Updated migration has a generated name. You can use ``--name`` to preserve the
+previous name.
```

Comment:
```suggestion
The updated migration will have a generated name; in order to preserve the existing name you must set it using ``--name``.
```

Suggestion^^ I think the wording flows better, others may have better wording 🤷‍♂️

Reply:
Updated, but not with exact wording :wink: